### ISSUE TITLE
Update or delete.

### ISSUE THREAD

**Issue by db00t** [staxrip/x265-encoding-tutorial#1]
This tutorial must be updated to todays GUI.

**Comment by mboyden** [comment]
Agreed; it is helpful post, yet things have moved around and it took some time to interpolate and find them in the latest GUI. I'd also love to see another set of tutorials that goes beyond the basics - showing how to use the various sets of tools and settings to address real-world issues encountered when re-encoding videos, and how to adjust them based on the particular video aspects. Grainy, dark, animation, horizontal pans that jerk or "crack", B&W, etc.
